2018-09-10-fotos-xcamp-2018.md

```diff
@@ -0,0 +1,10 @@
+---
+title:     Tolle Eindrücke und spannenden Teilnehmer beim XCamp 2018
+author:    Jens Bothmer
+published: 10.09.2018
+image:     
+tags:      XCamp2018, Sessions
+---
+Fotos und Eindrücke vom XCamp 2018 – Barcamp für digitale Innovation und agile Produktentwicklung findet ihr unter dem folgenden Link. Besonders danken wir [Tobias von Garnier](https://von-garnier.jimdo.com/), der für uns den Großteil der tollen Fotos geschossen hat. Seht einfach selbst und findet Euch gern wieder. Wir sehen uns auf dem XCamp 2019.
+
+<a class="fusion-button-text" target="_blank" href="https://xcamp.co/download/Fotos_XCamp18/">FOTOS ANSEHEN.</a>
```

Comment:
The `fusion-button-text` css class won't work here - it is specific for WordPress.
We should use the classes `button big` instead.